hooks/__init__.py

```diff
@@ -0,0 +1,23 @@
+"""KPOps pre-commit hooks"""
+import os
+from contextlib import contextmanager
+from pathlib import Path
+
+PATH_ROOT = Path(__file__).parents[1]
+
+# Taken from https://stackoverflow.com/a/24176022/11610149
+
+
+@contextmanager
+def cd(newdir: str | Path = PATH_ROOT):
```

Comment:
clever use of a contextmanager :) why not stick to pathlib, is the `os` module really necessary?

Reply:
I couldn't find how to change dir without `os`